scripts/exgfs_wave_prep.sh

```diff
@@ -224,16 +224,16 @@
               ;;
```

Comment:
Please revert these changes, this script is not involved w/the forecast and should remain as is for this PR.

Reply:
OK. I addressed these.